### PR TITLE
feat(doc): ldap tenant id and docker image

### PR DESCRIPTION
**Pull request by baptistemesta**
add it to the release note and add the new property `tenantId` and `BONITA_TENANT_ID` to the documenation

Relates to [RUNTIME-425](https://bonitasoft.atlassian.net/browse/RUNTIME-425)
Closes [RUNTIME-435](https://bonitasoft.atlassian.net/browse/RUNTIME-435)

### CODE REVIEW

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = LDAP synchronizer
 :description: Learn how to configure the LDAP Synchronizer. This tool updates Bonita Users and Groups database from the LDAP directory.
 
-Learn how to configure the LDAP Synchronizer. This tool updates Bonita Users and Groups database from the LDAP directory.
```
Comment (educhastenier) [inline modules/ROOT/pages/ldap-synchronizer.adoc]: revert this

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 2]
```diff
@@ -10,12 +10,12 @@ For Enterprise, Performance, Efficiency, and Teamwork editions only.
 
 The LDAP synchronization service keeps the Bonita organization information synchronized with an external LDAP directory, by creating, updating, or removing
 users and groups in the organization
-to match as closely as possible the user list attributes defined in LDAP. It does not modify the LDAP directory.
```
Comment (educhastenier) [inline modules/ROOT/pages/ldap-synchronizer.adoc]: revert this

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 3]
```diff
@@ -10,12 +10,12 @@ For Enterprise, Performance, Efficiency, and Teamwork editions only.
...
+to match as closely as possible the user list attributes defined in LDAP.It does not modify the LDAP directory.
 
 The LDAP Synchronizer connects to both the LDAP server and Bonita Engine to read the data from the LDAP server and modify the organization in the Bonita Engine database using the API.
 Both the Engine and the LDAP server must be running when the LDAP Synchronizer tool is run.
 
-The synchronizer program runs in the background. There is no user interface: all configuration settings are specified in files and
```
Comment (educhastenier) [inline modules/ROOT/pages/ldap-synchronizer.adoc]: revert this

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 4]
```diff
@@ -49,9 +49,10 @@ where `x.y.z` is the version of Bonita you are running.
 
 WARNING: Do not modify the Organization from the Bonita Portal while the tool is running, as this will cause a synchronization error.
 
-NOTE: Some use cases require additional configuration server side (for example to use directly the engine API). They are described xref:configure-client-of-bonita-bpm-engine.adoc[here].
+NOTE: Some use cases require additional configuration server side (for example to use directly the engine API).They are described xref:configure-client-of-bonita-bpm-engine.adoc[here].
```
Comment (educhastenier) [inline modules/ROOT/pages/ldap-synchronizer.adoc:53]: revert this

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 5]
```diff
@@ -197,17 +200,18 @@ There are five properties files:
...
 * bonita.properties defines the Bonita connection settings and specifies the account used for user synchronization (requires administration privileges).
 * ldap.properties defines the LDAP connection settings and specifies the account used for user browsing.
-* logger.properties provides the settings for the logger. Default settings should be fine for most uses.
+* logger.properties provides the settings for the logger.Default settings should be fine for most uses.
 * mapper.properties specifies the translation between Bonita and LDAP directory user attributes.
 * sync.properties defines the synchronization settings.
 
 All configurations files can be found in the conf directory.
 
-NOTE: to use a special character in a properties file, use the Unicode equivalent. For example, for `à` use \u00E0.
```
Comment (educhastenier) [inline modules/ROOT/pages/ldap-synchronizer.adoc]: revert this

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 6]
```diff
@@ -197,17 +200,18 @@ There are five properties files:
 
 * bonita.properties defines the Bonita connection settings and specifies the account used for user synchronization (requires administration privileges).
 * ldap.properties defines the LDAP connection settings and specifies the account used for user browsing.
-* logger.properties provides the settings for the logger. Default settings should be fine for most uses.
```
Comment (educhastenier) [inline modules/ROOT/pages/ldap-synchronizer.adoc]: revert this

**Review by educhastenier — CHANGES_REQUESTED** [review]
Revert some space removals

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 7]
```diff
@@ -1,7 +1,7 @@
 = LDAP synchronizer
 :description: Learn how to configure the LDAP Synchronizer. This tool updates Bonita Users and Groups database from the LDAP directory.
 
-Learn how to configure the LDAP Synchronizer. This tool updates Bonita Users and Groups database from the LDAP directory.
+Learn how to configure the LDAP Synchronizer.This tool updates Bonita Users and Groups database from the LDAP directory.
```
Comment (rbioteau) [inline modules/ROOT/pages/ldap-synchronizer.adoc]:
```suggestion
Learn how to configure the LDAP Synchronizer. This tool updates Bonita Users and Groups database from the LDAP directory.
```

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 8]
```diff
@@ -10,12 +10,12 @@ For Enterprise, Performance, Efficiency, and Teamwork editions only.
 
 The LDAP synchronization service keeps the Bonita organization information synchronized with an external LDAP directory, by creating, updating, or removing
 users and groups in the organization
-to match as closely as possible the user list attributes defined in LDAP. It does not modify the LDAP directory.
+to match as closely as possible the user list attributes defined in LDAP.It does not modify the LDAP directory.
```
Comment (rbioteau) [inline modules/ROOT/pages/ldap-synchronizer.adoc]:
```suggestion
to match as closely as possible the user list attributes defined in LDAP. It does not modify the LDAP directory.
```

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 9]
```diff
@@ -10,12 +10,12 @@ For Enterprise, Performance, Efficiency, and Teamwork editions only.
...
 
 The LDAP Synchronizer connects to both the LDAP server and Bonita Engine to read the data from the LDAP server and modify the organization in the Bonita Engine database using the API.
 Both the Engine and the LDAP server must be running when the LDAP Synchronizer tool is run.
 
-The synchronizer program runs in the background. There is no user interface: all configuration settings are specified in files and
+The synchronizer program runs in the background.There is no user interface: all configuration settings are specified in files and
```
Comment (rbioteau) [inline modules/ROOT/pages/ldap-synchronizer.adoc]:
```suggestion
The synchronizer program runs in the background. There is no user interface: all configuration settings are specified in files and
```

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 10]
```diff
@@ -197,17 +200,18 @@ There are five properties files:
 
 * bonita.properties defines the Bonita connection settings and specifies the account used for user synchronization (requires administration privileges).
 * ldap.properties defines the LDAP connection settings and specifies the account used for user browsing.
-* logger.properties provides the settings for the logger. Default settings should be fine for most uses.
+* logger.properties provides the settings for the logger.Default settings should be fine for most uses.
```
Comment (baptistemesta) [inline modules/ROOT/pages/ldap-synchronizer.adoc]:
```suggestion
* logger.properties provides the settings for the logger. Default settings should be fine for most uses.
```

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 11]
```diff
@@ -197,17 +200,18 @@ There are five properties files:
...
 * mapper.properties specifies the translation between Bonita and LDAP directory user attributes.
 * sync.properties defines the synchronization settings.
 
 All configurations files can be found in the conf directory.
 
-NOTE: to use a special character in a properties file, use the Unicode equivalent. For example, for `à` use \u00E0.
+NOTE: to use a special character in a properties file, use the Unicode equivalent.For example, for `à` use \u00E0.
```
Comment (baptistemesta) [inline modules/ROOT/pages/ldap-synchronizer.adoc]:
```suggestion
NOTE: to use a special character in a properties file, use the Unicode equivalent. For example, for `à` use \u00E0.
```

File: modules/ROOT/pages/ldap-synchronizer.adoc [hunk 12]
```diff
@@ -49,9 +49,10 @@ where `x.y.z` is the version of Bonita you are running.
 
 WARNING: Do not modify the Organization from the Bonita Portal while the tool is running, as this will cause a synchronization error.
 
-NOTE: Some use cases require additional configuration server side (for example to use directly the engine API). They are described xref:configure-client-of-bonita-bpm-engine.adoc[here].
+NOTE: Some use cases require additional configuration server side (for example to use directly the engine API).They are described xref:configure-client-of-bonita-bpm-engine.adoc[here].
```
Comment (baptistemesta) [inline modules/ROOT/pages/ldap-synchronizer.adoc]:
```suggestion
NOTE: Some use cases require additional configuration server side (for example to use directly the engine API). They are described xref:configure-client-of-bonita-bpm-engine.adoc[here].
```

**Pull request merged**